pyproject.toml

```diff
@@ -0,0 +1,8 @@
+[tool.black]
+line-length = 120
```

Comment:
I hate any `line-length` restriction since it prevents me from having used all my horizontal screen estate.
From my side the programmer should decide when it makes sense to break a line rather than enforcing it with something like that.

Do you know any reason why people like those `line-length` restrictions?

Reply:
I like it, because it keeps the whole project on a similar code-standard and it helps a lot when using side-by side diffs.
Vertical code is most of the time simpler to parse than horisontal code imo

an example (the first application of this rule in the code):
```                        
for d in day.get("dishes"):
    self.dishes.append(Dish(d.get("name"), d.get("prices"), d.get("ingredients", ""), d.get("dish_type"), date))
````
⬇️ 
````
for dish in day.get("dishes"):
    self.dishes.append(
        Dish(
            dish.get("name"),
            dish.get("prices"),
            dish.get("ingredients", ""),
            dish.get("dish_type"),
            date,
        ),
    )
````
120 Characters is imo quite a good compromise between readability and length..

https://softwareengineering.stackexchange.com/questions/604/is-the-80-character-limit-still-relevant-in-times-of-widescreen-monitors

I can remove this rule if you'd like

Reply:
OK. Good point. Thanks for the explanation. Never the less I'd say we remove this.
This does not mean we have to abandoned something like you have shown above since stuff like this make definitely sense. But I do no like any hard limit for line length since there are special cases where you need long lines, without having to add an additional line to exclude it from linting.